@Test public void should_fail_validating_class_with_two_custom_constructors() throws Exception {
  setExec(aptUtils -> {
    final TypeElement typeElement=aptUtils.elementUtils.getTypeElement(TestEntityWithNoPublicConstructor.class.getCanonicalName());
    final TypeName typeName=ClassName.get(TestEntityWithTwoCustomConstructors.class);
    beanValidator.validateConstructor(aptUtils,typeName,typeElement);
  }
);
  failTestWithMessage("Entity type 'info.archinnov.achilles.internals.sample_classes.parser.validator.TestEntityWithTwoCustomConstructors' should",TestEntityWithTwoCustomConstructors.class);
}
